doofinder-for-woocommerce/assets/js/wizard.js

```diff
@@ -1,117 +1,115 @@
-                ? window.screenLeft
-                : window.screenX;
-        const dualScreenTop =
-            window.screenTop !== undefined ? window.screenTop : window.screenY;
-        const width = window.innerWidth
-            ? window.innerWidth
-            : document.documentElement.clientWidth
-            ? document.documentElement.clientWidth
-            : screen.width;
-        const height = window.innerHeight
-            ? window.innerHeight
-            : document.documentElement.clientHeight
-            ? document.documentElement.clientHeight
-            : screen.height;
-        const systemZoom = width / window.screen.availWidth;
-        const left = (width - w) / 2 / systemZoom + dualScreenLeft;
-        const top = (height - h) / 2 / systemZoom + dualScreenTop;
-        const newWindow = window.open(
-            url,
-            title,
-            `
+  const popupCenter = ({ url, title, w, h }) => {
+    const dualScreenLeft =
+      window.screenLeft !== undefined ? window.screenLeft : window.screenX;
+    const dualScreenTop =
+      window.screenTop !== undefined ? window.screenTop : window.screenY;
+    const width = window.innerWidth
+      ? window.innerWidth
+      : document.documentElement.clientWidth
+      ? document.documentElement.clientWidth
+      : screen.width;
+    const height = window.innerHeight
+      ? window.innerHeight
+      : document.documentElement.clientHeight
+      ? document.documentElement.clientHeight
+      : screen.height;
+    const systemZoom = width / window.screen.availWidth;
+    const left = (width - w) / 2 / systemZoom + dualScreenLeft;
+    const top = (height - h) / 2 / systemZoom + dualScreenTop;
+    const newWindow = window.open(
+      url,
+      title,
+      `
         scrollbars=yes,
         width=${w / systemZoom},
         height=${h / systemZoom},
         top=${top},
         left=${left}
       `
-        );
+    );
 
-        if (window.focus) {
-            newWindow.focus();
-        }
+    if (window.focus) {
+      newWindow.focus();
+    }
 
-        return newWindow;
-    };
+    return newWindow;
+  };
 
-    const confirmLeavePage = (active = true) => {
-        if (active) {
-            window.onbeforeunload = () => "";
-        } else {
-            window.onbeforeunload = () => null;
-        }
-    };
+  const confirmLeavePage = (active = true) => {
+    if (active) {
+      window.onbeforeunload = () => "";
+    } else {
+      window.onbeforeunload = () => null;
+    }
+  };
 
-    $(".open-window").click((e) => {
-        const pageType = $(e.currentTarget).attr("data-type");
-        connectModal = popupCenter({
-            url: `${doofinderAdminPath}/plugins/${pageType}/wordpress?token=${doofinderConnectToken}`, // Change in future task endpoint name in doomanager.
-            title: "DoofinderConnect",
-            w: 600,
-            h: 700,
-        });
+  $(".open-window").click((e) => {
```

Comment:
This shorthand is deprecated since jQuery 3.3. https://api.jquery.com/click-shorthand/. A better option could be:

```suggestion
  $(".open-window").on("click", (e) => {
```